fdb-relational-core/src/test/java/com/apple/foundationdb/relational/recordlayer/PlanGenerationStackTest.java

```diff
@@ -162,7 +162,8 @@ public Stream<? extends Arguments> provideArguments(final ExtensionContext conte
                     Arguments.of(75, "with recursive c as (select * from t, c) select * from c", "recursive CTE does not contain non-recursive term"),
                     Arguments.of(76, "with recursive c1 as (select * from restaurant union all select * from c1) select * From c1", null),
                     Arguments.of(77, "with recursive c as (with recursive c1 as (select * from restaurant union all select * from c1) select * From c1 union all select * from restaurant, c) select * from c", null),
-                    Arguments.of(78, "with recursive c as (with c as (select * from restaurant) select * from c union all select * from restaurant) select * from c union all select * from restaurant", "ambiguous nested recursive CTE name")
+                    Arguments.of(78, "with recursive c as (with c as (select * from restaurant) select * from c union all select * from restaurant) select * from c union all select * from restaurant", "ambiguous nested recursive CTE name"),
+                    Arguments.of(79, "with recursive c1(name) as (select * from restaurant union all select * from c1) select * From c", "cte query has 7 column(s), however 1 aliases defined")
```

Comment:
I always approve of additional testing, but to make sure I didn't miss a code change, why an additional test here?

Reply:
Right, the additional test was to confirm the fix done here [0] in `SemanticAnalyzer`'s validation of the CTE columns, where the incorrect `%s` in the error message is replaced with `%d` as it is a placeholder for column count which is a number.

[0] https://github.com/FoundationDB/fdb-record-layer/pull/3117/files#diff-02328f5aa27f48c66a951cf05b4a1f0e866d2f3488f7471bfba6bf49c108cb4aR695